<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Explore</class>
 <widget class="QWidget" name="Explore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>730</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Thin</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/rescources/rescources/icons/Plus.png</normaloff>:/rescources/rescources/icons/Plus.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>379</width>
     <height>746</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>377</width>
      <height>744</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="SearchBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>379</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <zorder>scrollArea</zorder>
  <zorder>AddButton</zorder>
  <zorder>SearchBar</zorder>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
